### PR TITLE
Python: Port old experimental points-to based queries

### PR DESCRIPTION
**Pull request by RasmusWL**
I did not spend enough time on these to fully promote them, just took the minimal effort to move them off points-to so we can properly deprecate/remove that in the future :+1:

### CODE REVIEW

File: python/ql/src/experimental/Security/CWE-091/XsltConcept.qll [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+   * Extend this class to model new APIs. If you want to refine existing API models,
+   * extend `XsltExecution` instead.
+   */
+  abstract class Range extends DataFlow::Node {
+    /** Gets the argument that specifies the XSLT transformer. */
+    abstract DataFlow::Node getXsltArg();
+  }
+}
+
+// -----------------------------------------------------------------------------
```
Comment (yoff) [inline python/ql/src/experimental/Security/CWE-091/XsltConcept.qll:55]: From here seems to be converted modelling from `experimental.semmle.python.security.injection.XSLT`.

**Review by yoff — CHANGES_REQUESTED** [review]
I think it is possible to delete `python/ql/src/experimental/semmle/python/security/injection/XSLT.qll` as part of this?

There is also a suggested rename, but I am not sure it is important.

It would have been nice if [the first commit](https://github.com/github/codeql/pull/13990/commits/91edde72c4855491edc1593ffdd02c675ee0ffb4) had been broken up so that files do not both move (get merged) and change. But in this case, through a setup with two windows, it was not too bad to compare bits of one file containing new modelling with deleted files containing old modelling...

File: python/ql/src/experimental/Security/CWE-074/TemplateInjectionCustomizations.qll [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+private import semmle.python.dataflow.new.RemoteFlowSources
+private import semmle.python.dataflow.new.BarrierGuards
+private import TemplateConstructionConcept
+
+/**
+ * Provides default sources, sinks and sanitizers for detecting
+ * "template injection"
+ * vulnerabilities, as well as extension points for adding your own.
+ */
+module SqlInjection {
```
[the remark this answers is not in the document]
Comment (RasmusWL) [reply python/ql/src/experimental/Security/CWE-074/TemplateInjectionCustomizations.qll]: Done in 39e2b13

File: python/ql/src/experimental/Security/CWE-091/XsltConcept.qll (hunk 1 above)
Comment (RasmusWL) [reply python/ql/src/experimental/Security/CWE-091/XsltConcept.qll:55]: yep :+1: I've removed the `XSLT.qll` file now :+1: (
0dca8a5)

**Comment by RasmusWL** [comment]
> It would have been nice if [the first commit](https://github.com/github/codeql/pull/13990/commits/91edde72c4855491edc1593ffdd02c675ee0ffb4) had been broken up so that files do not both move (get merged) and change. But in this case, through a setup with two windows, it was not too bad to compare bits of one file containing new modelling with deleted files containing old modelling...

My bad :neutral_face:

I've fixed things up, so should be good to go now :+1:

**Review by yoff — APPROVED** [review]
LGTM, thanks for getting this done.

**Review by yoff — APPROVED** [review]
LGTM

**Pull request merged**